app.js

```diff
@@ -1,25 +1,29 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const logger = require("morgan");
+const cors = require("cors");
 
-const contactsRouter = require('./routes/api/contacts')
 
-const app = express()
+const contactsRouter = require("./routes/api/contacts");
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
+const app = express();
 
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
+// const fs = require("fs");
+// const path = require("path");
+// const accessLogFile = fs.createWriteStream(path.join(__dirname, "access.log"), { flags: "a" });
 
-app.use('/api/contacts', contactsRouter)
+app.use(logger('dev'));
+// app.use(logger('dev'), { stream: accessLogFile });
+app.use(cors());
+app.use(express.json());
+
+app.use("/api", contactsRouter);
```

Comment:
```suggestion
app.use("/api/contacts", contactsRouter);
```
Tu dodaj contacts, a w contactsRouter usuń ze ścieżki to słowo

Reply:
Tak miałam na początku, ale potem pomyślałam, że być może w dalszej części będziemy tę aplikację rozbudowywać o inne endpointy i dlatego zostawiłam tylko '/api' w app.js, a resztę wrzuciłam w contacts router.
Poprawione.